README.md

```diff
@@ -166,6 +167,68 @@ To do so, we are implementing anonymous diagnostic data collection.
 We will never collect any user sensitive data, only integration related
 warnings and errors. See related [developers discussion](https://github.com/leikoilja/ha-google-home/discussions/116).
 
+## Troubleshooting
+
+### Collecting useful log data
+
+Here are the steps to generate useful log data:
+
+1. Temporary log level change.
+   1. Visit [Home Assistant Developer Services Tool](https://my.home-assistant.io/redirect/developer_services/).
+   2. Choose `Logger: Set level` from the **Service** menu.
+   3. Under **Service data** paste the following:
+      ```yaml
+      custom_components.google_home: debug
+      glocaltokens: debug
+      ```
+   4. Click **Call Service**.
+2. Read the log information.
+   1. Visit [Home Assistant Logs](https://my.home-assistant.io/redirect/logs/).
+   2. Click **Load Full Home Assistant Log**.
+   3. Look for all `google_home` entries.
+   4. Also look for all `glocaltokens` entries.
+3. Requesting help with the log information.
+   1. Copy the log entries.
+   2. Paste them into a discussion forum or bug report between the following two lines:
+      > \```
+      >
+      > \```
```

Comment:
Also do we really want to go in such details?

Reply:
I think it should be short and on point, so it is easier to understand even if you are not tech savvy 😊

Reply:
@DurgNomis-drol is it yes or no? 🙂

Reply:
Sorry, no we should not go into details 😊

Reply:
Gents, @DurgNomis-drol, @KapJI some of your requests have been fulfilled in the latest push 😅